src/main/java/model/Figure.java

```diff
@@ -0,0 +1,21 @@
+package model;
+
+public abstract class Figure implements Area, Draw {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public double getArea() {
+        return 0;
+    }
+
+    public void drawing() {
+        System.out.println("This is figure!");
+    }
```

Comment:
we can remove the implementation of these methods from the Figure class, because class is abstract